keps/sig-apps/2255-pod-cost/README.md

```diff
@@ -55,191 +49,216 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
+  - No.
+
+
+* **Will enabling / using this feature result in any new calls to cloud
+  provider?**
+  - No.
+
+
+* **Will enabling / using this feature result in increasing size or count
+  of the existing API objects?**
+  Describe them providing:
+  - API type(s): Pod annotation
+  - Estimated increase in size: Size of a new annotation
+  - Estimated amount of new objects: new annotation for potentially every existing Pod
+
+
+* **Will enabling / using this feature result in increasing time taken by any
+  operations covered by [existing SLIs/SLOs][]?**
+  - It should have negligible impact on scale-down latency since we are 
+    adding an additional sorting key.
+
+
+* **Will enabling / using this feature result in non-negligible increase of
+  resource usage (CPU, RAM, disk, IO, ...) in any components?**
+  - No.
+
+## Implementation History
+- 2021-01-13: Initial KEP submitted as provisional
+- 2021-01-15: KEP promoted to implementable
 
 
+<!--
+
 ### Rollout, Upgrade and Rollback Planning
 
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
-  - As the feature is a simple annoation, the worst what could happen is that either the annotation is lost or ignored. In the worst case, a pod with a higher priority gets deleted before a pod with a lower priority.
 
 
 * **What specific metrics should inform a rollback?**
-  - None
 
 
 * **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
-  - Was tested. Behaviour change in both directions, as expected.
 
 
 * **Is the rollout accompanied by any deprecations and/or removals of features,
   APIs, fields of API types, flags, etc.?**
-  - No. However, the exact same pod annotation string cannot be used for any other purposes.
 
 
 ### Monitoring requirements
 
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can an operator determine if the feature is in use by workloads?**
-  - Search for pod annotations with the exact same pod-cost annotation string.
 
 
 * **What are the SLIs (Service Level Indicators) an operator can use to
   determine the health of the service?**
-  - A pod with a lower pod-cost annotation in a Deployment gets deleted first on a scale-down event.
 
 
 * **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
-  - All pods with a lower pod-cost annotation in a Deployment are deleted first on a scale-down event.
 
 * **Are there any missing metrics that would be useful to have to improve
   observability if this feature?**
-  - N/A
 
 ### Dependencies
 
 _This section must be completed when targeting beta graduation to a release._
 
 * **Does this feature depend on any specific services running in the cluster?**
-  - The feature requires the existing of the kube-controller-manager and the ability and permissions to set pod annotations.
-
-
-### Scalability
```

Comment:
Why not keep this? I think the status-quo is to leave the questions if they are not answered yet.

Reply:
sounds strange to have questions not answered, but I reverted the change.